box_embeddings/parameterizations/__init__.py

```diff
@@ -1,3 +1,5 @@
 from .box_tensor import TBoxTensor
 from .box_tensor import BoxTensor
 from .delta_box_tensor import MinDeltaBoxTensor
+from .sigmoid_box_tensor import SigmoidBoxTensor
```

Comment:
We should either call them SigmoidBoxTensor and TanhBoxTensor or SigmoidActivatedBoxTensor and TanhActivatedBoxTensor (I vote for the first one) because apart from the fact that the input for the box tensor comes from the corresponding non-linearities, there is no other difference between these two classes.

Reply:
Replaced TanhActivatedBoxTensor with TanhBoxTensor.